src/systems/sprites/sprite.vert

```diff
@@ -0,0 +1,28 @@
+#version 300 es
+
+uniform mat4 modelViewMatrix;
+uniform mat4 projectionMatrix;
+
+in vec4 mvCol0;
+in vec4 mvCol1;
+in vec4 mvCol2;
+in vec4 mvCol3;
+in vec3 a_vertices;
+in vec2 a_uvs;
+out vec2 v_uvs;
+in float a_hubs_EnableSweepingEffect;
+flat out float v_hubs_EnableSweepingEffect;
+in vec2 a_hubs_SweepParams;
+flat out vec2 v_hubs_SweepParams;
+out vec3 hubs_WorldPosition;
+
+void main() {
+  mat4 mv = mat4(mvCol0, mvCol1, mvCol2, mvCol3);
+  gl_Position = projectionMatrix * modelViewMatrix * mv * vec4(a_vertices, 1.0);
```

Comment:
traditionally the modelview matrix is for going from model space to view space. I assume mv is just from going from model to world space, then you want to take that into view space. So what you really want here is just the inverse camera matrix (view matrix) and the model world matrix. Since the sprite's worldMatrix ends up being identity anyway, this is the same thing as modelViewMatrix, albiet with a slightly more confusing name.

I think renderObject computes modelViewMatrix for every object regardless of if we use it or not, so its makes sense to keep as is, though I would maybe change the name of `mv` to make it look less like the view matrix is being applied twice.